From 0fd61785f56c2785b471e1d2dd1071a480380c3f Mon Sep 17 00:00:00 2001
From: Yuxi Sun <b36102@freescale.com>
Date: Wed, 12 Oct 2011 12:17:02 +0800
Subject: [PATCH] ENGR00159738 v4l2: correct wrong parameter when V4l2 set window size

Correct wrong parameter when call ipu_csi_set_window_size function

Signed-off-by: Yuxi Sun <b36102@freescale.com>
(cherry picked from commit c1cb33e5cbebb979967f74eecf55efe6a83884ab)
---
 drivers/media/video/mxc/capture/mxc_v4l2_capture.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/media/video/mxc/capture/mxc_v4l2_capture.c b/drivers/media/video/mxc/capture/mxc_v4l2_capture.c
index ded1839..c030a39 100644
--- a/drivers/media/video/mxc/capture/mxc_v4l2_capture.c
+++ b/drivers/media/video/mxc/capture/mxc_v4l2_capture.c
@@ -1594,7 +1594,7 @@ static int mxc_v4l_open(struct file *file)
 		pr_debug("On Open: Input to ipu size is %d x %d\n",
 				cam_fmt.fmt.pix.width, cam_fmt.fmt.pix.height);
 		ipu_csi_set_window_size(cam->crop_current.width,
-					cam->crop_current.width,
+					cam->crop_current.height,
 					cam->csi);
 		ipu_csi_set_window_pos(cam->crop_current.left,
 					cam->crop_current.top,
-- 
1.5.4.4

